README.md

```diff
@@ -1 +1,20 @@
-# blog
\ No newline at end of file
+# .layer blog
+
+The objective behind this blog is to make it easy for the community members (and anybody else) to promote their work and share their thoughts and ideas related to the field of data science/machine learning.
+
+## New post tutorial
+
+Here is a quick tutorial that explains few steps to follow in order to contribute to this blog.
+
+First, to be reproducible through the time, and mainly through all different users, this repo should be **independent** of any specific code compilations (R, Python, etc). For example, in a data analysis, the graphics, values and models should already have been calculated by the user and the results included in a compiled document such as a .md or .html extension.
```

Comment:
First, to maximize reproducibility through the time, and for all different users, the repo should be **independent** of any specific code compilations (R, Python, etc).

Reply:
ou "should stay"?